zproject/urls.py

```diff
@@ -460,6 +460,9 @@
     url(r'^archive/streams/(?P<stream_id>\d+)/topic/(?P<topic_name>[^/]+)$',
         zerver.views.archive.archive,
         name='zerver.views.archive.archive'),
+    url(r'^archive/(?P<stream_id>\d+)/topics$',
```

Comment:
This URL should probably be ` url(r'^archive/streams/(?P<stream_id>\d+)/topics$'`

(and we probably want to add another commit changing `'^archive/streams/(?P<stream_id>\d+)/topic/` to pluralize `topics` as well for consistency in naming).